sphinx/writers/latex.py

```diff
@@ -1459,8 +1466,14 @@ def depart_entry(self, node):
             else:  # a bottom part of multirow cell
                 self.table.col += nextcell.width
                 self.body.append('&')
-                if nextcell.width > 1:  # use \multicolumn for wide multirow cell
-                    self.body.append('\\multicolumn{%d}{l|}{}\\relax ' % nextcell.width)
+                if nextcell.width == 1:
+                    # insert suitable strut for equalizing row heights in multirow
+                    self.body.append('\\sphinxtablestrut{%d}' % nextcell.cell_id)
+                else:
+                    # use \multicolumn for wide multirow cell
+                    self.body.append('\\multicolumn{%d}{l|}'
+                                     '{\\sphinxtablestrut{%d}}\\relax ' %
```

Comment:
same remark about ``\\relax `` here. It comes from earlier code but appears unneeded. I will push a commit to remove it and the other one at L#1391